### ISSUE TITLE
Found 24335 images belonging to 1 classes.

### ISSUE THREAD

**Issue by suri199507** [jupyter/notebook#4506]
I am using google colab  for  image classification problem ,  I have  upload  the data file in google drive imported to colab ,  I want  to do image augmentation  for purpose ,  i did this 

[from keras.preprocessing.image import ImageDataGenerator
train_generator = train_datagen.flow_from_directory(
    directory=r"train-scene classification",
    target_size=(98, 98),
    color_mode="rgb",
    batch_size=32,
    class_mode="categorical",
    shuffle=True,
    seed=42
) ](url)

I have 6 classes of images in one folder but got only 1 class, please help me how to do this augmentation.
My files like these on colab
[ls/content/
adc.json     sample_data/                   'train-scene classification'/
 sample.csv   sample_submission_CH2mq5Z.csv   train.zip](url)

[ls/content/train-scene classification'/
test.csv  train/  train.csv
](url)

**Comment by minrk** [comment]
I think you might be in the wrong place. This looks like a keras question. Good luck!
